README.md

```diff
@@ -1,39 +1,14 @@
-Pull Request Template
-=====================
+attn: Pull Request Template
```

Comment:
(But don't get me started on the discordance between our name and logo. #ohwell)